frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -122,13 +134,14 @@ export default class DashboardGrid extends Component {
     const initialSize = DEFAULT_CARD_SIZE;
     const minSize = visualization.minSize || DEFAULT_CARD_SIZE;
     return {
-      i: dashcard.id,
+      i: String(dashcard.id),
```

Comment:
does `react-grid-layout` require string ids?

Reply:
Yes, take a look at the code block in their docs [here](https://github.com/react-grid-layout/react-grid-layout#grid-item-props)